src/test/cuecontrol_test.cpp

```diff
@@ -148,8 +148,6 @@ TEST_F(CueControlTest, LoadTrackWithIntroEndAndOutroStart) {
 
     loadTrack(pTrack);
 
-    // Default Cue is at 0.0
-    EXPECT_DOUBLE_EQ(0.0, m_pCuePoint->get());
```

Comment:
Why did you remove this? I guess this still succeeds.

Reply:
It was failing and isn't relevant to what this is testing. The comment is also wrong; the default cue point is determined by AnalyzerSilence.

Reply:
OK.